packages/flutter_tools/lib/src/tester/flutter_tester.dart

```diff
@@ -233,3 +234,21 @@ class _FlutterTesterDeviceLogReader extends DeviceLogReader {
 
   void addLine(String line) => _logLinesController.add(line);
 }
+
+/// A fake port forwarder that doesn't do anything. Used by flutter tester
+/// where the VM is running on the same machine and does not need ports forwarding.
+class _NoopPortForwarder extends DevicePortForwarder {
+  @override
+  Future<int> forward(int devicePort, {int hostPort}) {
+    if (hostPort != null && hostPort != devicePort)
+      throw 'Forwarding to a different port is not supported by flutter tester';
+    return new Future<int>.value(devicePort);
+  }
+
+  // TODO: implement forwardedPorts
```

Comment:
`// TODO(dantup):` in order to give people the best idea of who to contact for more information

Reply:
Oops, this was stale - this is implemented.